lib/class-convertkit-api.php

```diff
@@ -376,14 +376,14 @@ public function get_sequences() {
 			return $response;
 		}
 
-		// If no sequences exist, return WP_Error.
+		// If no sequences exist, log that no sequences exist and return a blank array.
 		if ( ! isset( $response['courses'] ) ) {
 			$this->log( 'API: get_sequences(): Error: No sequences exist in ConvertKit.' );
-			return new WP_Error( 'convertkit_api_error', $this->get_error_message( 'get_sequences_none' ) );
```

Comment:
Quick question, if there are no courses, would courses still be set in the response?

Reply:
Fixed.